### PR TITLE
Support "skip rendering" mode in amp-viewer-integration/highlight-handler.js

### PR DESCRIPTION
**Pull request by yunabe**
This pull request should be merged after https://github.com/ampproject/amphtml/pull/15791. There is only one new commit in this PR (https://github.com/ampproject/amphtml/pull/15950/commits/465138d4b23033f4f2b1a24649ba06e9b7c89eb5).

### CODE REVIEW

File: extensions/amp-viewer-integration/0.1/highlight-handler.js [hunk 1]
```diff
@@ -48,10 +56,20 @@ const NUM_ALL_CHARS_LIMIT = 1500;
 
 /**
  * TextRange represents a text range.
- * @typedef {{sentences: !Array<string>}}
+ * If doNothing is true, checks whether highlighted texts are found but do not
+ * show highlighted texts to users.
+ * @typedef {{sentences: !Array<string>, doNothing: boolean}}
```
Comment (alanorozco) [inline extensions/amp-viewer-integration/0.1/highlight-handler.js]: What's the purpose of checking without highlighting?

Comment (yunabe) [reply extensions/amp-viewer-integration/0.1/highlight-handler.js]: Replied privately.

Comment (alanorozco) [reply extensions/amp-viewer-integration/0.1/highlight-handler.js]: Please rename to `skipRendering`.

Comment (yunabe) [reply extensions/amp-viewer-integration/0.1/highlight-handler.js]: Done

**Comment by yunabe** [comment]
Let me know if there is something I can do to help you merge this PR.

**Pull request merged**